build/commands/lib/config.js

```diff
@@ -255,6 +255,7 @@ const Config = function () {
   this.useBraveHermeticToolchain = this.rbeService.includes('.brave.com:')
   this.brave_services_key_id = getEnvConfig(['brave_services_key_id']) || ''
   this.service_key_aichat = getEnvConfig(['service_key_aichat']) || ''
+  this.meldApiKey = getEnvConfig(['meld_api_key']) || ''
```

Comment:
Refer to my other comment re proxying the API endpoint, passing api keys by command-line flags makes it difficult to rotate keys in future.